homeassistant/components/knx/light.py

```diff
@@ -180,9 +180,11 @@ def should_poll(self):
     @property
     def brightness(self):
         """Return the brightness of this light between 0..255."""
-        if not self.device.supports_brightness:
-            return None
-        return self.device.current_brightness
+        if self.device.supports_brightness:
+            return self.device.current_brightness
+        if self.device.supports_color and self._hsv_color:
+            return round(self._hsv_color[-1] / 100 * 255)
```

Comment:
Looks like we need to check for `None` here, since `self._hsv_color` can return `None`.

Reply:
```suggestion
            return round(self._hsv_color[-1] / 100 * 255) if self._hsv_color else None
```